anago

```diff
@@ -278,8 +282,25 @@ rev_version_base () {
   logecho -n "Formatting $version_file: "
   logrun -s gofmt -s -w $version_file
   logrun git add $version_file
-  logecho -n "Committing $version_file: "
-  logrun -s git commit -m "Kubernetes version ${RELEASE_VERSION[$label]}"
+  if [[ -f $staging_file ]]; then
+    logrun cp $version_file $staging_file
+    logrun git add $staging_file
+  fi
+
+  #########################################################################
+  # NOTE: To avoid a significant time sink from running update-all.sh as
+  #       well as avoiding running the master-only doc stub creator
+  #       update-generated-docs.sh, we simply and quickly modify the files that
+  #       are dependent on a $version_file change
+  #########################################################################
+  for f in ${swagger_files[*]}; do
+    sed -i 's,"version": "'${VER_REGEX[release]}'","version": ${RELEASE_VERSION[$label]},g' $f
```

Comment:
Does this line work? If I'm reading it correctly, it looks like `${RELEASE_VERSION[$label]}` is going to be inserted literally. Also does it need to be surrounded by double quotes since it's a JSON file?